cost/azure/rightsize_sql_instances/azure_rightsize_sql_instances.pt

```diff
@@ -46,6 +46,13 @@ parameter "param_email" do
   label "Email addresses"
   description "Email addresses of the recipients you wish to notify"
 end
+parameter "param_automatic_rightsize_instance" do
+    type "string"
+    label "Automatically rightsize sql instance"
+    description "When this value is set to yes this policy will automatically take the rightsize sql instance action"
```

Comment:
Consider changing to:
When this value is set to yes this policy will automatically rightsize the sql instance